docs/examples/localstack.md

```diff
@@ -0,0 +1,134 @@
+# LocalStack
```

Comment:
file should be under modules?

Reply:
I was tempted to do it, but then the changes would be much bigger. I'm working in a PR that separate modules and examples at the generator level

Reply:
I will reorganise the docs in a follow-up PR, as it needs to update the code generation tool